raw-svo.setup.lua

```diff
@@ -244,6 +244,12 @@ signals.gmcpcharitemsremove:connect(function()
   end
 end)
 signals.gmcpcharvitals      = luanotify.signal.new()
+signals.gmcpcharvitals:connect(function()
+  if not gmcp.Char.Vitals.charstats then return end
+  for index, val in ipairs(gmcp.Char.Vitals.charstats) do
```

Comment:
How does the structure of charstats look like? There's a possibility of stats.battlerage being set, and then reset on a subsequent iteration because the Rage: string isn't found.